OneCommunityGlobal/HGNRest
# Imran edit summary submit requirement others

# Description
Gives the user permission to change the requirement to the user to submit a summary.

## Related PRS (if any):
To test this backend PR you need to checkout the [PR#2242](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2242)  frontend PR.


## Main changes explained:
- added updateSummaryRequirements permission
- added permission check for updateSummaryRequirements in profileController
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. Follow the frontend [PR#2242](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2242) to test


## Note:
Include the information the reviewers need to know.
